README.md

```diff
@@ -83,128 +122,132 @@ Install with your preferred package manager:
     },
 }
 ```
+
 </details>
 
 ## Usage
 
-| Command | Description |
-|---|---|
-| MCstart | Selects the word under cursor and starts listening for the actions. In visual mode it acts like `MCvisual` |
-| MCvisual | Selects the last visual mode selection and starts listening for the actions. |
-| MCpattern | Prompts for a pattern and selects every match in the buffer. |
-| MCvisualPattern | Prompts for a pattern and selects every match in the visual selection. |
-| MCunderCursor | Selects the char under cursor and starts listening for the actions. |
-| MCclear | Clears all the selection. |
+| Command         | script                                       | Description                                                                                                |
+|-----------------|----------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| MCstart         | require("multicursors").start()              | Selects the word under cursor and starts listening for the actions. In visual mode it acts like `MCvisual` |
+| MCvisual        | require("multicursors").search_visual()      | Selects the last visual mode selection and starts listening for the actions.                               |
+| MCpattern       | require("multicursors").new_pattern()        | Prompts for a pattern and selects every match in the buffer.                                               |
+| MCvisualPattern | require("multicursors").new_pattern_visual() | Prompts for a pattern and selects every match in the visual selection.                                     |
+| MCunderCursor   | require("multicursors").new_under_cursor()   | Selects the char under cursor and starts listening for the actions.                                        |
+| MCclear         | require("multicursors").exit()               | Clears all the selection.                                                                                  |
```

Comment:
```suggestion
```
The api isn't stable now, and i don't want to expose things that maybe gonna break in future.